src/app/zap-templates/zcl/data-model/chip/joint-fabric-administrator-cluster.xml

```diff
@@ -0,0 +1,82 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidPublicKey" value="0x01"/>
+    <item name="InvalidICAC" value="0x02"/>
+  </enum>
+
+  <enum name="StatusCodeEnum" type="enum8">
+    <cluster code="0x0753"/>
+     <item name="JF_VID_NOT_VERIFIED" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Administrator</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_ADMINISTRATOR_CLUSTER</define>
+    <description>Joint Fabric Administrator Cluster.</description>
+
+    <attribute side="server" code="0x0000" name="AdministratorFabricIndex" define="ADMINISTRATOR_FABRIC_INDEX" type="fabric_idx" default="0">
+      <description>AdministratorFabricIndex</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+
+    <command source="client" code="0x00" name="ICACCSRRequest" response="ICACCSRResponse" optional="false">
+      <description>ICACCSRRequest</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x01" name="ICACCSRResponse" optional="false">
+      <description>ICACCSRResponse</description>
+      <arg name="ICACCSR" type="octet_string" length="600"/>
+    </command>
+
+    <command source="client" code="0x02" name="AddICAC" response="ICACResponse" optional="false">
+      <description>AddICAC</description>
+      <arg name="ICACValue" type="octet_string" length="400"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x03" name="ICACResponse" optional="false">
+      <description>ICACResponse</description>
+      <arg name="StatusCode" type="ICACResponseStatusEnum"/>
+    </command>
+
+    <command source="client" code="0x04" name="OpenJointCommissioningWindow" optional="false">
+      <description>OpenJointCommissioningWindow</description>
+      <arg name="CommissioningTimeout" type="int16u"/>
+      <arg name="PAKEPasscodeVerifier" type="octet_string"/>
+      <arg name="Discriminator" type="int16u"/>
+      <arg name="Iterations" type="int32u"/>
+      <arg name="Salt" type="octet_string" length="32"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x08" name="AnnounceJointFabricAdministrator" optional="false">
```

Comment:
The spec says this is a server-to-client command, so a response command.  What is it a response to?  Certainly it's not marked as a server-to-client command here nor is it a response to anything.

Anyway, this XML is clearly not matching the spec.  No idea which one is wrong, or if both are.

Reply:
fixed the spec. side, the directionality of the commands was broken: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11584/